core/include/detray/propagator/actor_chain.hpp

```diff
@@ -26,16 +26,15 @@ namespace detray {
 /// It can hold both simple actors, as well as an actor with its observers.
 /// The states of the actors need to be passed to the chain in an external tuple
 ///
-/// @tparam tuple_t tuple type used to resolve the actor types
 /// @tparam actors_t the types of the actors in the chain.
-template <template <typename...> class tuple_t = dtuple, typename... actors_t>
+template <typename... actors_t>
```

Comment:
Necessary? It is going to require change traccc and any other repositories using detray

Reply:
This reduces the degree of templating in detray and will result in a very minor changes in traccc. I think it is OK